cmd/spire-server/cli/x509/mint.go

```diff
@@ -182,3 +170,55 @@ func (c *mintCommand) Run(ctx context.Context, env *common_cli.Env, serverClient
 func ttlToSeconds(ttl time.Duration) int32 {
 	return int32((ttl + time.Second - 1) / time.Second)
 }
+
+type MintResult struct {
```

Comment:
It doesn't seem that we need to export this, should be unexported.

Reply:
You're right. Thank you for the review @amartinezfayo 😄

Reply:
238ceef